scripts/lxc-to-lxd

```diff
@@ -9,6 +9,80 @@ import subprocess
 import sys
 
 
+supported_keys = [
+        'lxc.arch',
+        'lxc.pts',
+        # 'lxc.tty',
+        # 'lxc.devttydir',
+        # 'lxc.kmsg',
+        'lxc.aa_profile',
+        'lxc.aa_allow_incomplete',
+        # 'lxc.se_context',
+        'lxc.cgroup',
+        # 'lxc.id_map',
+        'lxc.loglevel',
+        # 'lxc.logfile',
+        'lxc.mount.entry',
+        'lxc.mount.auto',
+        'lxc.mount',
+        # 'lxc.rootfs.mount',
+        # 'lxc.rootfs.options',
+        # 'lxc.rootfs.backend',
+        'lxc.rootfs',
```

Comment:
Also, every container will have this set. So if we kick it, we will not import any container at all. :)